drracket/drracket/private/syncheck/blueboxes-gui.rkt

```diff
@@ -317,7 +317,7 @@
       (set! to-invalidate (currently-drawn-bluebox-region))
       (maybe-invalidate))
     
-    (define pref-changed-callback (λ (x y) (invalidate-blue-box-region)))
+    (define (pref-changed-callback x y) (invalidate-blue-box-region))
```

Comment:
This is another one where I wrote it that way on purpose to emphasize that we've not got a method so that we're holding onto a reference to the object to keep the callback registration alive.